### PR TITLE
feat: main pod for account

### LINKED ISSUE #46: default POD value
at some places such as POD creation, the default value for the pod name is `main` but at other commands require pod parameter such as `fdp-cli directory create [dirName]` does not have this default value.
It seems a bit inconsistent.
The best solution would be IMO if the config file would store the default value for pod parameters per account

### PR DESCRIPTION
**Pull request by IgorShadurin**
Close #46

### CODE REVIEW

**Comment by nugaon** [comment]
please rebase with master first

File: src/command/file/file-command.ts [hunk 1]
```diff
@@ -7,7 +7,6 @@ export class FileCommand extends RootCommand {
     key: 'account',
     alias: 'a',
     description: 'Name of the account',
-    required: { when: 'quiet' },
```
Comment (nugaon) [inline src/command/file/file-command.ts:10]: keep quiet mode for non-interactive sessions, because of this, leave it here for now.

Comment (nugaon) [reply src/command/file/file-command.ts:10]: same at `pod`

**Review by nugaon — COMMENTED** [review]
Nice work!

Please read my comments below and add description about main pod and account in general, for what purposes it can be used.

**Review by nugaon — APPROVED** [review]
LGTM!

**Comment by tfius** [comment]
looks nice

**Pull request merged**